src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowCrudSpec.groovy

```diff
@@ -155,19 +153,19 @@ class FlowCrudSpec extends HealthCheckSpecification {
         * we will not test such case
         */
         data << flowsWithoutTransitSwitch + flowsWithTransitSwitch + singleSwitchFlows
-        flow = data.flow as FlowRequestV2
-        srcDstStr = "src:${topology.find(flow.source.switchId).hwSwString}->dst:${topology.find(flow.destination.switchId).hwSwString}"
+        expectedFlowEntity = data.expectedFlowEntity as FlowExtended
```

Comment:
As we discussed in a huddle, some data tests fail here due to 400 BAD REQUEST when some tg is marked as inactive

Reply:
The issue was connected to the absence of the available traffgens for traffic passes (one switch flow).
According to the old logic, in case of any error during traffic verification, it was caught and ignored. With the new implementation and 'healthy' topology (active traffgens), the traffic verification for each created flow was available.
As a result, a new logic was slightly changed to be flexible to environment configuration.